browser/search_engines/search_engine_provider_util.cc

```diff
@@ -25,7 +25,7 @@ namespace brave {
 
 void SetBraveAsDefaultPrivateSearchProvider(PrefService* prefs) {
   auto data = TemplateURLPrepopulateData::GetPrepopulatedEngine(
-      prefs, TemplateURLPrepopulateData::PREPOPULATED_ENGINE_ID_BRAVE);
+      prefs, TemplateURLPrepopulateData::PREPOPULATED_ENGINE_ID_GOOGLE);
```

Comment:
this id is set as brave search so is private tab using brave search again now? please check

Reply:
NHI bhaiya, its still the same for the pivate tab, 

PREPOPULATED_ENGINE_ID_GOOGLE is google for the private tab

Reply:
ye change revert karke check karna ek baar

Reply:
checked bhaiya, with my build

Reply:
revert karke nahi chal rha kya?

Reply:
checking bhaiya